### ISSUE TITLE
Should /tournamenter-obr/ verify if you logged first?

### ISSUE THREAD

**Issue by jnydutra** [ivanseidel/tournamenter-obr#1]
When I access this page i redirected direct to Pontuador, even if I not logged into tournamenter app

**Comment by ivanseidel** [comment]
It will allow you to see it, but you wont be able to save or load
teams/scores.

Thats the expected behavior

**Comment by jnydutra** [comment]
Yeah! But isn't better to redirect to /login page? Suppose that an judge mark all scores before verify if it logged? Then it will lost match information when change to login page?

**Comment by ivanseidel** [comment]
Thats a good point. However, we cannot do a redirect on the server since
all files are hosted statically by plugins (they get merged in a "public"
folder during initialization).

But, it could be done on the frotend, just after loading the page. If the
user is not logged in, then we can make a warning or redirect to login.
That would be easier and would allow the session to be checked not only
once during load, but at every minute after that maybe?

Can you make this feature?

**Comment by jnydutra** [comment]
I will do! Thanks @ivanseidel :D
